apps/tangle-dapp/data/DelegationsPayouts/usePayouts.ts

```diff
@@ -281,7 +292,7 @@ export default function usePayouts(
       isMounted = false;
       sub?.unsubscribe();
     };
-  }, [address]);
+  }, [address, cachedPayouts, setCachedPayouts]);
```

Comment:
Won't this cause infinite `useEffect` loop, since it will re-trigger on `cachedPayouts `, but then we're changing it inside by using `setCachedPayouts`? I think we should be using `setCachedPayouts((prev) => ...` here instead to avoid any possibility of that.